ztp/source-crs/ClusterVersionStatus.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterVersion
```

Comment:
We already have the ClusterVersion CR in source-crs, why do we need a new copy?

Reply:
I would suggest to keep two ClusterVersion CRs.
1. ClusterVersionUpdateGraph.yaml only contains channel & upstream. This can be used to workaround the CVO issue https://bugzilla.redhat.com/show_bug.cgi?id=2055314 if needed.
2. ClusterVersion.yaml contains the channel, upstream with default values and desiredUpdate.version and status check.

We can easily remove the ClusterVersionUpdateGraph.yaml if the CVO fix merged to 4.9

Reply:
I have updated the PR to split the upstream & channel change from the version change separate CRs ordered by the argoCD wave number.